cobalt/site/docs/reference/starboard/modules/14/event.md

```diff
@@ -399,5 +399,5 @@ of microseconds to wait before calling the `callback` function. Set `delay` to
 #### Declaration
 
 ```
-SbEventId SbEventSchedule(SbEventCallback callback, void *context, SbTime delay)
+SbEventId SbEventSchedule(SbEventCallback callback, void *context, int64_t delay)
```

Comment:
I'm worried about those here. Technically Starboard 14 shouldn't be changing , and none of the functions should either. @gbournou can you have a look - should we have #ifdef-gated it ?

E.g. for SB16 use definition with int64, for previous versions SbTime ?